### PR TITLE
[OTS-432]: swift 3 migration

### PR DESCRIPTION
**Pull request by jvalli**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/opentok/annotation-acc-pack/pull/76" target="_blank">ReviewNinja

**Comment by albertoacn** [comment]
:+1 for ninja-review

File: ios/AnnotationAccPackKit/OTAnnotationAccPackKit/AnnotationOnScreenViewController.swift [hunk 1]
```diff
@@ -7,38 +7,38 @@
...
         
-        let statusButton = UIButton(frame: CGRectMake(0, 0, CGRectGetWidth(UIScreen.mainScreen().bounds), 20))
-        statusButton.addTarget(self, action: #selector(AnnotationOnScreenViewController.stopAnnotation), forControlEvents: .TouchUpInside)
+        let statusButton = UIButton(frame: CGRect(x: 0, y: 0, width: UIScreen.main.bounds.width, height: 20))
+        statusButton.addTarget(self, action: #selector(AnnotationOnScreenViewController.stopAnnotation), for: .touchUpInside)
         statusButton.backgroundColor = UIColor(red: 118.0/255.0, green: 206.0/255.0, blue: 31.0/255.0, alpha: 1.0)
-        statusButton.setTitle("You are annotating your screen, Tap here to dismiss", forState: .Normal)
-        statusButton.titleLabel!.font = UIFont.systemFontOfSize(12.0)
-        annotationOverContentViewController.view.addSubview(statusButton)
+        statusButton.setTitle("You are annotating your screen, Tap here to dismiss", for: UIControlState())
```
Comment (Lucashuang0802) [inline ios/AnnotationAccPackKit/OTAnnotationAccPackKit/AnnotationOnScreenViewController.swift]: Is it a mistake of the converter?

File: ios/AnnotationAccPackKit/OTAnnotationAccPackKit/AnnotationManualViewController.swift [hunk 2]
```diff
@@ -39,15 +39,15 @@ class AnnotationManualViewController: UIViewController {
         let p3 = OTAnnotationPoint(x: 126, andY: 18)
         let p4 = OTAnnotationPoint(x: 119, andY: 16)
         let p5 = OTAnnotationPoint(x: 144, andY: 28)
-        let path = OTAnnotationPath(points: [p1, p2, p3, p4, p5], strokeColor: nil)
-        screenShareView.annotationView.addAnnotatable(path)
+        let path = OTAnnotationPath(points: [p1!, p2!, p3!, p4!, p5!], stroke: nil)
+        screenShareView.annotationView.add(path)
```
Comment (Lucashuang0802) [inline ios/AnnotationAccPackKit/OTAnnotationAccPackKit/AnnotationManualViewController.swift]: Let's do `let p5: OTAnnotationPoint = OTAnnotationPoint(x: 144, andY: 28)` then remove `!`

File: ios/AnnotationAccPackKit/OTAnnotationAccPackKit/AnnotationManualViewController.swift [hunk 3]
```diff
@@ -39,15 +39,15 @@ class AnnotationManualViewController: UIViewController {
...
         
         let p6 = OTAnnotationPoint(x: 160, andY: 16)
         let p7 = OTAnnotationPoint(x: 160, andY: 20)
         let p8 = OTAnnotationPoint(x: 160, andY: 24)
         let p9 = OTAnnotationPoint(x: 160, andY: 26)
         let p10 = OTAnnotationPoint(x: 160, andY: 30)
-        let path2 = OTAnnotationPath(points: [p6, p7, p8, p9, p10], strokeColor: UIColor.redColor())
-        screenShareView.annotationView.addAnnotatable(path2)
+        let path2 = OTAnnotationPath(points: [p6!, p7!, p8!, p9!, p10!], stroke: UIColor.red)
+        screenShareView.annotationView.add(path2)
```
Comment (Lucashuang0802) [inline ios/AnnotationAccPackKit/OTAnnotationAccPackKit/AnnotationManualViewController.swift]: Let's do let p5: OTAnnotationPoint = OTAnnotationPoint(x: 144, andY: 28) then remove !

File: ios/AnnotationAccPackKit/OTAnnotationAccPackKit/AnnotationOnScreenViewController.swift (hunk 1 above)
Comment (jvalli) [inline ios/AnnotationAccPackKit/OTAnnotationAccPackKit/AnnotationOnScreenViewController.swift]: updated

File: ios/AnnotationAccPackKit/OTAnnotationAccPackKit/AnnotationManualViewController.swift (hunk 2 above)
Comment (jvalli) [inline ios/AnnotationAccPackKit/OTAnnotationAccPackKit/AnnotationManualViewController.swift]: updated

File: ios/AnnotationAccPackKit/OTAnnotationAccPackKit/AnnotationManualViewController.swift (hunk 3 above)
Comment (jvalli) [inline ios/AnnotationAccPackKit/OTAnnotationAccPackKit/AnnotationManualViewController.swift]: updated

**Comment by Lucashuang0802** [comment]
👍

**Pull request merged**